packages/components/src/test/testHelperMocks.ts

```diff
@@ -1,17 +1,16 @@
 import { Map } from 'immutable';
 import mock, { proxy } from 'xhr-mock';
 
-import { initUnitTests } from '../internal/test/testHelpers';
-
 import { initDomainPropertiesMocks, initQueryGridMocks, initUserPropsMocks } from './mock';
+import { initQueryGridState } from '../internal/global';
 
 /**
  * Use this method in beforeAll() for your jest tests and you'll have full access
  * to all of the same mock API responses we use in storybook.
  */
 export function initUnitTestMocks(extraMocks?: Array<() => void>, metadata?: Map<string, any>): void {
     window['__react-beautiful-dnd-disable-dev-warnings'] = true;
-    initUnitTests(metadata);
+    initQueryGridState(metadata);
```

Comment:
nit: In a brief scan I don't see any methods that hand in `metadata` to `initUnitTestMocks()`. Could this call to `initQueryGridState` be removed as well?